ospfd/ospf_abr.c

```diff
@@ -1788,6 +1788,28 @@ void ospf_abr_task(struct ospf *ospf)
 		zlog_debug("ospf_abr_task(): Stop");
 }
 
+/* If there's redistribution configured, we need to refresh external
+ * LSAs in order to install Type-7 and flood to all NSSA Areas
+ */
+static void ospf_abr_nssa_redist_task(struct ospf *ospf)
```

Comment:
Here function name  resembles , it is specific to ABR. But this is applicable to even non-ABRs right ? and more over this is specific to ASBR functionality , So I feel  moving this function to  ospf_asbr.c file is more meaningful.

Reply:
Moved